parquet.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import sublime
+import sublime_plugin
+
+import os
+import subprocess
+
+
+"""
+Parquet package for Sublime Text, with python tooling.
+https://github.com/yuj/sublime-parquet
+
+Requires one of the following:
+python: (https://github.com/ktrueda/parquet-tools) (preferred)
+java: (https://github.com/Parquet/parquet-mr/tree/master/parquet-tools) v1.12.0
+"""
+
+
+COMMAND_NOT_FOUND_MSG = """'parquet-tools' not found.\n
+Make sure python is in your PATH and install with 'pip install parquet-tools'
+"""
+PYTHON_COMMAND_LINE = "parquet-tools csv {0}"
```

Comment:
This is a bit out of scope, but I've been thinking to use something like `parquet-tools csv --head 1000000 {0}` to limit the number of lines to say 1 million rows to prevent ST from exploding :)

Reply:
So the python port doesn't support that keyword argument - maybe in a pyarrow/fastparquet-based implementation we could have a `max_lines` user settable flag or something similar to cap it if you're just trying to do inspection?

Reply:
Wait.  Are we talking about the same Python "parquet-tools"?   I can see the `--head` or `-n` option for csv command:

`$ parquet-tools csv -h
usage: parquet-tools csv [-h] [--columns COLUMNS] [--head HEAD] [--awsprofile AWSPROFILE] FILE [FILE ...]

Cat parquet as csv style.

positional arguments:
  FILE                  The parquet file to read. e.g. ./target.parquet or s3://bucket-name/target.parquet or s3://bucket-name/*

optional arguments:
  -h, --help            show this help message and exit
  --columns COLUMNS, -c COLUMNS
                        Show only the given column, can be specified more than once. e.g. --columns email,name
  --head HEAD, -n HEAD  Show only head record(default:infinity)
  --awsprofile AWSPROFILE
                        awscli profile in ~/.aws/credentials. You use this option when you read parquet file on s3.`

Reply:
I like your idea that users could set this limit themselves.

Reply:
@dogversioning  Anyways, as I said above, the number of lines limit thing is outside the scope, so let's not worry about it in this PR. We can always enhance it in the future.

I am going to merge this PR as is, and go from there.

Thanks for your contribution!